src/Core/Translation/Storage/Provider/ThemeCatalogueLayersProvider.php

```diff
@@ -247,6 +247,7 @@ private function getModulesTranslations(string $locale): MessageCatalogue
     {
         $moduleCatalogue = new MessageCatalogue($locale);
         $modules = $this->getModulesFromTranslations($locale);
```

Comment:
Maybe it's better to do the `array_unique` at the end of `getModulesFromTranslations` function.
What do you think about it?